### PR TITLE
Table version of the MT subset script

### PR DESCRIPTION
**Pull request by MattWellie**
Adds a second script, which copies a Hail Table. Mostly copied from the MatrixTable subset script, but removes all references to samples/genotypes

### CODE REVIEW

**Review by annacuomo — APPROVED** [review]

File: scripts/subset_hail_table.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+if __name__ == '__main__':
+
+    logging.basicConfig(
+        level=logging.INFO,
+        format='%(asctime)s %(levelname)s %(module)s:%(lineno)d - %(message)s',
+        datefmt='%Y-%M-%d %H:%M:%S',
+        stream=sys.stderr,
+    )
+
+    parser = ArgumentParser()
```
Comment (illusional) [inline scripts/subset_hail_table.py:140]: We generally use click for CLI, I think it's a bit cleaner. Not a big deal for this PR, but for new scripts :)

**Review by illusional — COMMENTED** [review]
If you provide no params, this won't filter the table though right? Could we ensure at least one filter is provided to avoid a straight copy?

**Comment by MattWellie** [comment]
@illusional There are times when a full copy would be useful, especially for smaller tables. Definitely something we should be mindful of in terms of storage costs if people are able to generate massive copies

**Comment by MattWellie** [comment]
Thanks @illusional, linting issues resolved. 

I'm going to hit merge here as @lgruen is at the team-leads meeting, but I'll calendar something to discuss this another time (as this + the MT copying script give users ability to copy mad amounts of data around, which should maybe be limited, or at least discussed)

**Pull request merged**